demisto_sdk/commands/content_graph/objects/integration_script.py

```diff
@@ -40,9 +43,13 @@ def get_supported_native_images(
         self, marketplace: MarketplaceVersions, ignore_native_image: bool = False
     ) -> List[str]:
         if marketplace == MarketplaceVersions.XSOAR and not ignore_native_image:
-            return ScriptIntegrationSupportedNativeImages(
-                _id=self.object_id,
-                docker_image=self.docker_image,
-                native_image_config=file_to_native_image_config(),
-            ).get_supported_native_image_versions(get_raw_version=True)
+            try:
+                return ScriptIntegrationSupportedNativeImages(
+                    _id=self.object_id,
+                    docker_image=self.docker_image,
+                    native_image_config=file_to_native_image_config(),
+                ).get_supported_native_image_versions(get_raw_version=True)
+            except FileNotFoundError:
+                logger.debug(f"The {NATIVE_IMAGE_FILE_NAME} file could not be found.")
```

Comment:
this _could_ be raised with another file, hiding the real cause for the failure. Please replace the `try-except` with `if not Path(...).exists(): return []`

Reply:
done